### PR TITLE
Update Mainnet Branch

### PR DESCRIPTION
**Pull request by mnaamani**
Keeping mainnet uptodate from latest updates on master

### CODE REVIEW

File: bin/node/Cargo.toml [hunk 1]
```diff
@@ -3,7 +3,7 @@ authors = ['Joystream contributors']
 build = 'build.rs'
 edition = '2018'
 name = 'joystream-node'
-version = '8.0.0'
+version = '7.4.1'
```
Comment (Lezek123) [inline bin/node/Cargo.toml:6]: Looks like the changes from https://github.com/Joystream/joystream/pull/4428 were reverted

File: bin/utils/chain-spec-builder/Cargo.toml [hunk 2]
```diff
@@ -3,7 +3,7 @@ authors = ['Joystream contributors']
 build = 'build.rs'
 edition = '2018'
 name = 'chain-spec-builder'
-version = '8.0.0'
+version = '6.8.0'
```
Comment (Lezek123) [inline bin/utils/chain-spec-builder/Cargo.toml:6]: Same as above

File: runtime/Cargo.toml [hunk 3]
```diff
@@ -4,7 +4,7 @@ edition = '2018'
 name = 'joystream-node-runtime'
 # Follow convention: https://github.com/Joystream/substrate-runtime-joystream/issues/1
 # {Authoring}.{Spec}.{Impl} of the RuntimeVersion
-version = '12.1000.0'
+version = '11.3.0'
```
Comment (Lezek123) [inline runtime/Cargo.toml:7]: Same as above

File: runtime/src/lib.rs [hunk 4]
```diff
@@ -146,8 +146,8 @@ pub use content::MaxNumber;
 pub const VERSION: RuntimeVersion = RuntimeVersion {
     spec_name: create_runtime_str!("joystream-node"),
     impl_name: create_runtime_str!("joystream-node"),
-    authoring_version: 12,
-    spec_version: 1000,
+    authoring_version: 11,
+    spec_version: 3,
```
Comment (Lezek123) [inline runtime/src/lib.rs:150]: Same as above

File: runtime/src/lib.rs [hunk 5]
```diff
@@ -1023,21 +1023,12 @@ parameter_types! {
...
-#[cfg(feature = "runtime-benchmarks")]
-parameter_types! {
-    pub const CouncilSize: u32 = 3;
-}
-
-#[cfg(feature = "playground-runtime")]
-parameter_types! {
-    pub const CouncilSize: u32 = 1;
-}
-
```
Comment (Lezek123) [inline runtime/src/lib.rs:1040]: Hmm, it looks like for some reason this merge is reverting all changes introduced in https://github.com/Joystream/joystream/pull/4428

**Review by Lezek123 — CHANGES_REQUESTED** [review]

File: runtime/src/lib.rs (hunk 5 above)
Comment (Lezek123) [reply runtime/src/lib.rs:1040]: Possibly related to https://github.com/Joystream/joystream/commit/dabaa27f3ed5efb406d7560e7205ebe1dc0aee3f

Comment (mnaamani) [reply runtime/src/lib.rs:1040]: Hmm thats odd. Thanks for spotting this. I'll merge locally and open a separate PR in this case. Perhaps I will see merge conflicts and make the right choice.

Comment (mnaamani) [reply runtime/src/lib.rs:1040]: fixed in https://github.com/Joystream/joystream/pull/4465

**Comment by mnaamani** [comment]
superceded by https://github.com/Joystream/joystream/pull/4465

**Pull request closed without merging**